bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx112-en-features.html

```diff
@@ -0,0 +1,255 @@
+    {% call split(
+      block_class='mzp-l-split-center-on-sm-md',
+      body_class='mzp-l-split-v-center',
+      media_class='mzp-l-split-h-center',
+      image=resp_img('img/firefox/whatsnew/whatsnew112-en/pocket-discover-high-res.png',
+        optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+      ),
+    ) %}
+      <h2>Discover the best of the web</h2>
+      <p>Pocket editors comb the web for entertaining and informative reads from trusted publications like Wired, Popular Science, Fast Company and more.</p>
+      <p>
+        {{ pocket_fxa_button(
+          entrypoint=entrypoint,
+          button_text='Activate Pocket in Firefox',
+          class_name='mzp-c-button mzp-t-secondary',
+          is_button_class=False,
+          optional_parameters={
+            's': _entrypoint,
+            'utm_campaign': _utm_campaign,
+            'utm_content': 'signup-fxa-primary'
+          },
+          optional_attributes={
+            'data-cta-text': 'Activate Pocket in Firefox',
+            'data-cta-type': 'button',
+          })
+        }}
+      </p>
+    {% endcall %}
+
+  {% elif variant == '4' %}
+
+    {# v4 - Pocket in first position #}
+    {% call split(
+      block_class='mzp-l-split-center-on-sm-md',
+      body_class='mzp-l-split-v-center',
+      media_class='mzp-l-split-h-center',
+      image=resp_img('img/firefox/whatsnew/whatsnew112-en/privacy-header.jpg',
+        optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+      ),
+    ) %}
+      <h2>Protect what’s important all year long</h2>
+      <p>Protect your location & activity using Mozilla VPN.</p>
+      <p>With the flip of a switch, Mozilla VPN works in the background to protect your entire device.</p>
+      <p>
+        {{ vpn_product_referral_link(
+          referral_id='whatsnew-112-en-v4-features-vpn-first',
+          link_text='Get Mozilla VPN',
+          class_name='mzp-t-secondary',
+          optional_attributes= {
+              'data-cta-text' : 'Get Mozilla VPN',
+              'data-cta-type' : 'button'
+          })
+        }}
+      </p>
+    {% endcall %}
+
+  {% else %}
+
+    {# v5 or v6 - ETP in first position #}
+    {% call split(
+      block_class='mzp-l-split-center-on-sm-md',
+      body_class='mzp-l-split-v-center',
+      media_class='mzp-l-split-h-center',
+      image=resp_img('img/firefox/whatsnew/whatsnew112-en/shield.png',
+        srcset={ 'img/firefox/whatsnew/whatsnew112-en/shield-high-res.png': '2x' },
+        optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+      ),
+    ) %}
+      <h2>See which trackers Firefox blocked for you this week</h2>
+      <p>We use Enhance Tracking Protection to automatically protect your privacy while you browse.</p>
+      <p>
+        <a class="mzp-c-button mzp-t-secondary js-open-about-protections" href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?{{ params }}"  data-cta-text="View my protections" data-cta-type="button">
```

Comment:
Nit: You're already firing a GA event in JS when this is clicked, so I think you can remove `data-cta-text="View my protections" data-cta-type="button"`?